Dnn.AdminExperience/Extensions/Content/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SqlConsole/scripts/jspdf.js

```diff
@@ -10,51 +10,67 @@
       return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj;
     };
 
-    /** @preserve
-     * jsPDF - PDF Document creation from JavaScript
-     * Version 1.2.61 Built on 2016-06-13T12:22:26.808Z
-     *                           CommitID 5f0008057e
-     *
-     * Copyright (c) 2010-2014 James Hall <james@parall.ax>, https://github.com/MrRio/jsPDF
-     *               2010 Aaron Spike, https://github.com/acspike
-     *               2012 Willow Systems Corporation, willow-systems.com
-     *               2012 Pablo Hess, https://github.com/pablohess
-     *               2012 Florian Jenett, https://github.com/fjenett
-     *               2013 Warren Weckesser, https://github.com/warrenweckesser
-     *               2013 Youssef Beddad, https://github.com/lifof
-     *               2013 Lee Driscoll, https://github.com/lsdriscoll
-     *               2013 Stefan Slonevskiy, https://github.com/stefslon
-     *               2013 Jeremy Morel, https://github.com/jmorel
-     *               2013 Christoph Hartmann, https://github.com/chris-rock
-     *               2014 Juan Pablo Gaviria, https://github.com/juanpgaviria
-     *               2014 James Makes, https://github.com/dollaruw
-     *               2014 Diego Casorran, https://github.com/diegocr
-     *               2014 Steven Spungin, https://github.com/Flamenco
-     *               2014 Kenneth Glassey, https://github.com/Gavvers
-     *
-     * Licensed under the MIT License
-     *
-     * Contributor(s):
-     *    siefkenj, ahwolf, rickygu, Midnith, saintclair, eaparango,
-     *    kim3er, mfo, alnorth, Flamenco
-     */
-
-    /**
-     * Creates new jsPDF document object instance.
-     *
-     * @class
-     * @param orientation One of "portrait" or "landscape" (or shortcuts "p" (Default), "l")
-     * @param unit        Measurement unit to be used when coordinates are specified.
-     *                    One of "pt" (points), "mm" (Default), "cm", "in"
-     * @param format      One of 'pageFormats' as shown below, default: a4
-     * @returns {jsPDF}
-     * @name jsPDF
-     */
-    var jsPDF = function (global) {
-        'use strict';
-
-        var pdfVersion = '1.3',
-            pageFormats = { // Size in pt of various paper formats
+/** @preserve
+ * jsPDF - PDF Document creation from JavaScript
+ * Version ${versionID}
+ *                           CommitID ${commitID}
+ *
```

Comment:
I'm concerned that these placeholders indicate you're pulling a source version instead of a distribution version.   Can you pleases switch to https://cdn.jsdelivr.net/npm/jspdf@1.5.3/dist/jspdf.min.js

Reply:
OK will do thanks